### PR TITLE
Change reference flag to "reference-file"

### PR DESCRIPTION
**Pull request by btmonier**
## Description
Hot fix to change the `reference` flag to `reference-file`. Also removes unused imports.

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [x] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

**Review by zrm22 — APPROVED** [review]

**Review by tcasstevens — APPROVED** [review]

File: src/main/kotlin/net/maizegenetics/phgv2/cli/AlignAssemblies.kt [hunk 1]
```diff
@@ -62,7 +60,7 @@ class AlignAssemblies : CliktCommand(help="Align assemblies using anchorwave") {
             }
         }
 
-    val reference by option(help = "Full path to reference fasta file")
+    val referenceFile by option(help = "Full path to reference fasta file")
```
Comment (lynnjo) [inline src/main/kotlin/net/maizegenetics/phgv2/cli/AlignAssemblies.kt:63]: The AlignAssembliesTest file must also be updated as well as the FullPipelineIT test

Comment (btmonier) [reply src/main/kotlin/net/maizegenetics/phgv2/cli/AlignAssemblies.kt:63]: Yep, see prior commit

Comment (lynnjo) [reply src/main/kotlin/net/maizegenetics/phgv2/cli/AlignAssemblies.kt:63]: Still need the FullPIpelineIT test changed.

Comment (btmonier) [reply src/main/kotlin/net/maizegenetics/phgv2/cli/AlignAssemblies.kt:63]: Updated

**Review by lynnjo — APPROVED** [review]

**Pull request merged**